Dockerfile

```diff
@@ -152,6 +152,22 @@ RUN if [ ! -z "${PG_PROMETHEUS}" ]; then \
         done; \
     fi
 
+ARG TIMESCALE_PROMETHEUS=
+# build and install the pg_prometheus extension
+RUN if [ ! -z "${TIMESCALE_PROMETHEUS}" ]; then \
+        mkdir -p /build \
+        && git clone https://github.com/timescale/timescale-prometheus /build/timescale_prometheus \
+        && set -e \
+        && for pg in ${PG_VERSIONS}; do \
```

Comment:
This Docker image contains multiple PostgreSQL binaries, 11 and 12 currently.
The idiomatic way we currently install is by looping over the versions.
I could rip it out, but then with 13 support (I'm hoping we'll have that soon) we'll have to put it back in.

Reply:
Will having both/three binaries make it easier to run upgrade scripts between versions? Like upgrade image version, but still start PG11, ???, shutdown, start PG12, run upgrade. Or the idea is to utilize some other mechanism?

Reply:
To upgrade, it is required to have binaries of the current `$PGDATA` directory and the newer `$PGDATA` directory, so yes, that's the main reason to stick multiple binaries in.